models/Element/Traits/KeyValueTrait.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @category   Pimcore
+ * @package    Element
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Model\Element\Traits;
+
+trait KeyValueTrait
+{
+    /**
+     * @var array
+     */
+    protected $_temporaryValues;

```

Comment:
This property needs to be excluded here: https://github.com/pimcore/pimcore/blob/163f95e61ac97b8d5997932827ba93cc85755b0b/lib/Model/AbstractModel.php#L217